serverless.yml

```diff
@@ -519,7 +519,7 @@ functions:
       - schedule:
           name: ${self:service}-${self:custom.stage}-index-vault-harvests
           description: 'index onchain harvests for all vaults'
-          rate: rate(20 minutes)
+          rate: rate(2 minutes)
```

Comment:
do we rly want it that often?

Reply:
no, forgot to push my fixes after self review. thanks!